dll/win32/shell32/CDefaultContextMenu.cpp

```diff
@@ -499,6 +499,7 @@ CDefaultContextMenu::AddStaticContextMenusToMenu(
     UINT fState;
     UINT cIds = 0, indexFirst = *pIndexMenu;
 
+    mii = {0};
```

Comment:
This initialization can be merged with old line 496.

--

Old line 505 was already redundant with old line 515. It becomes redundant with the added initialization too.
Old line 515 remains redundant with old line 553.
I suggest to remove them.

Reply:
Not sure what all this about old lines is, but you can move this initialization directly to the declaration.

Optionally, you could also move the sizeof into the declaration like this:
`MENUITEMINFOW mii = { sizeof(mii) };

Reply:
Thanks for these comments, I was trying to adjust the old code as little as possible because y'all know how style changes are an uphill battle in this project. I followed Mark's suggestion.

Reply:
> I suggest to remove them.

Submitted as #6768.